pages/Sidebar.js

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react';
+
+const Sidebar = () => {
+  const [selectedItem, setSelectedItem] = useState('About');
+
+  const handleItemClick = (item) => {
+    setSelectedItem(item);
+  };
+
+  return (
+    <div className='fixed top-0 left-0 h-screen w-1/5 box-border bg-[#081f2e] '>
+      <div className='flex-row h-70 text-center my-4'>
+        <img src='/images/joemon1.jpg' alt="Image Description" className="w-[10rem] h-[10rem] mx-auto rounded-full " />
+        <h3>Joemon Mathew</h3>
+      </div>
+      <div></div>
+      <ul className='flex flex-col items-center h-full  mr-1 space-y-5 bg-[#020107] box-border '>
+        <li className='mt-5'>
+          <a
+            onClick={() => handleItemClick('About')}
```

Comment:
If implemented properly we won't need this click handler

Reply:
actually that wasnt for navigating instead i used that for the sidebar selection. 
anyway ill update the PR now and please let me know more about the same